### ISSUE TITLE
fal script not executing (Could not read dbt sources artifact)

### ISSUE THREAD

**Issue by quinnhornblow** [fal-ai/fal#545]
Hey team,

I've noticed that some of my fal scripts are not executing. It took me a while to realize because it doesn't raise an obvious error.
Basically I call ```fal flow run --select [script.py]``` and the process runs in a couple of seconds, except that the script is never called. I can tell because there is a print statement that doesn't get output in the console, plus the outputs are not updated.

There is one line in the log ```Could not read dbt sources artifact``` which might be hinting at the error.

See the full log below with some personal details omitted:

```
fal flow run --select dry_weather_records_excl_tide

File 'fal\analysis\tidal_response.sql' was generated from 'analysis\tidal_response.py'.
Please do not modify it directly. We recommend committing it to your repository.
File 'fal\master_tables\calculated_flow.sql' was generated from 'master_tables\calculated_flow.py'.
Please do not modify it directly. We recommend committing it to your repository.
File 'fal\site_stats\hirds_parameters.sql' was generated from 'site_stats\hirds_parameters.py'.
Please do not modify it directly. We recommend committing it to your repository.
File 'fal\site_stats\kpis_twi.sql' was generated from 'site_stats\kpis_twi.py'.
Please do not modify it directly. We recommend committing it to your repository.
File 'fal\site_stats\rainfall_depth_frequency.sql' was generated from 'site_stats\rainfall_depth_frequency.py'.
Please do not modify it directly. We recommend committing it to your repository.
File 'fal\wet_dry_weather\dry_weather_records_excl_tide.sql' was generated from 'wet_dry_weather\dry_weather_records_excl_tide.py'.
Please do not modify it directly. We recommend committing it to your repository.
20:50:34  Found 34 models, 14 tests, 0 snapshots, 0 analyses, 167 macros, 4 operations, 12 seed files, 0 sources, 0 exposures, 0 metrics
Could not read dbt sources artifact
Running command: dbt run --threads 1 --project-dir C:\_\_\_\_\_\_\_--select dry_weather_records_excl_tide
20:50:39  Running with dbt=1.1.1
20:50:39  Found 34 models, 14 tests, 0 snapshots, 0 analyses, 167 macros, 4 operations, 12 seed files, 0 sources, 0 exposures, 0 metrics
20:50:39
20:50:44  Concurrency: 1 threads (target='dev')
20:50:44
20:50:45
20:50:45  Running 4 on-run-end hooks
20:50:45  1 of 4 START hook: xus_dbt.on-run-end.0 ..................................... [RUN]
20:50:45  1 of 4 OK hook: xus_dbt.on-run-end.0 ........................................ [GRANT in 0.07s]
20:50:45  2 of 4 START hook: xus_dbt.on-run-end.1 ..................................... [RUN]
20:50:45  2 of 4 OK hook: xus_dbt.on-run-end.1 ........................................ [GRANT in 0.06s]
20:50:45  3 of 4 START hook: xus_dbt.on-run-end.2 ..................................... [RUN]
20:50:46  3 of 4 OK hook: xus_dbt.on-run-end.2 ........................................ [GRANT in 0.06s]
20:50:46  4 of 4 START hook: xus_dbt.on-run-end.3 ..................................... [RUN]
20:50:46  4 of 4 OK hook: xus_dbt.on-run-end.3 ........................................ [GRANT in 0.06s]
20:50:46
20:50:46
20:50:46  Finished running 4 hooks in 6.85s.
```

fal version = 0.5.2
dbt-postgres version = 1.2.1

**Comment by turbo1912** [comment]
Hi @quinnhornblow thanks for the bug report! 

Can you also tell us a little bit about your `schema.yml` file? Do you have python scripts as well as fal python models?

**Comment by quinnhornblow** [comment]
Hi @turbo1912 ,

My schema.yml is very basic.

```
version: 2
models:
  - name: dry_weather_records_excl_tide
    description: "All records that are free of rain and tide influence"
```

The fal-python model 'dry_weather_records_excl_tide.py' calls another function inside a utils.py file. Similar to the setup used in [jaffle-shop-fal](https://github.com/fal-ai/jaffle_shop_with_fal) repo.
The documentation seems to have changed slightly since I got this workflow going but it is intended to be a fal-python model not associated with a dbt model.

**Comment by quinnhornblow** [comment]
Interestingly I was able to re-run the fal-python model successfully by first deleting the generated sql file. Although haven't been able to repeat this successfully (only worked once). But note the +12hr time shift from the successful log, 17:03 is the correct local time.

```
05:03:25  Finished running 4 hooks in 0 hours 0 minutes and 6.69 seconds (6.69s).
17:03:26 | Starting fal run for following models and scripts: 
(model: models\wet_dry_weather\dry_weather_records_excl_tide.py)
```

**Comment by quinnhornblow** [comment]
Fixed with version 0.6 release.
